packages/docs/docs/docs/advanced.md

```diff
@@ -138,6 +138,13 @@ data = encodeCall(
 
 await exampleToken.sendTransaction( {data, from: owner} );
 ```
+## Safety checks
+
+The ZeppelinOS CLI performs a series of safety checks in some of its commands with the purpose of strengthening the security of your application. For example, the `zos add` command can detect if a contract has a `constructor`, or contains usages of `selfdestruct` or `delegatecall`. Below is a list of checks made, along with the reasoning behind why they are considered to be security risks:
+
+* constructor check: Proxied contracts should use initializer functions instead of constructors. For more info, see "Initializers vs. constructors" in this page.
```

Comment:
You can link here to the section, right? I'm not sure how to make a relative link, but I suppose it's possible.